src/olympia/addons/indexers.py

```diff
@@ -105,6 +105,16 @@ def get_mapping(cls):
                     'boost': {'type': 'float', 'null_value': 1.0},
                     'default_locale': {'type': 'keyword', 'index': False},
                     'description': {'type': 'text', 'analyzer': 'snowball'},
+                    'featured_for': {
+                        'type': 'nested',
+                        'properties': {
+                            'application': {'type': 'byte'},
+                            'locales': {
+                                'type': 'keyword',
+                                'null_value': 'ALL',
```

Comment:
oh man, I think that explains my concerns above. `null` means "all languages" which certainly makes this mapping very confusing to read unless you know that specific fact. Worth a comment imho.

Reply:
I thought changing it to be `ALL` was less confusing 😆